lib/modules/manager/gitlabci-include/extract.ts

```diff
@@ -80,11 +80,12 @@ function extractDepFromIncludeComponent(
   }
   const endpointUrl = parseUrl(endpoint);
   if (endpointUrl && endpointUrl.hostname !== componentReference.fqdn) {
+    const registryUrl = `https://${componentReference.fqdn}`;
```

Comment:
This will obviously not work if GitLab is served on another port or via HTTP (let's hope nobody does this 😄), but GitLab will run in the same issue...